app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,17 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         android:name=".FocusApplication">
+
+        <provider
+            android:name="android.support.v4.content.FileProvider"
```

Comment:
Wow, it's annoying that we have to create a FileProvider for this and can't just share the file:/// uri to the already public file. :(